pkg/contexts/ocm/accessmethods/relativeociref/README.md

```diff
@@ -0,0 +1,56 @@
+
+# Access Method `relativeOciReference` - OCI Artifact Access in OCI-registry-based OCM Repository
+
+
+### Synopsis
+
+```
+type: relativeOciReference/v1
+```
+
+Provided blobs use the following media type:
+
+- `application/vnd.oci.image.manifest.v1+tar+gzip`: OCI image manifests
+- `application/vnd.oci.image.index.v1+tar.gzip`: OCI index manifests
+
+Depending on the repository appropriate docker legacy types might be used.
+
+The artifact content is provided in the [Artifact Set Format](../../../oci/repositories/ctf/README.md#artifact-set-archive-format).
```

Comment:
No description of the Artifact Set Format at this link location.

```suggestion
The artifact content is provided in the [Artifact Set Format](../../../oci/repositories/ctf/formatspec.md#artifact-set-archive-format).
```

A README.md should also be added to the pkg/contexts/oci/repositories/artifactset directory.